jsx/Form.js

```diff
@@ -134,6 +134,162 @@ var FormElement = React.createClass({
   }
 });
 
+/**
+ * Search Component
+ * React wrapper for a searchable dropdown
+ */
+class SearchElement extends React.Component {
```

Comment:
I should have said this in the last review, but can we name it something like "SearchableDropdown"? SearchElement sounds like the type of textbox you see in the menu of some pages, not a dropdown that has autocomplete suggestions

Reply:
@driusan done